src/main/resources/org/opentripplanner/apis/transmodel/schema.graphql

```diff
@@ -325,6 +325,8 @@ type Leg {
   fromPlace: Place!
   "Generalized cost or weight of the leg. Used for debugging."
   generalizedCost: Int
+  "The leg's headway in seconds"
+  headway: Int
```

Comment:
@Bartosz-Kruba  Does the Transmodel API have a Duration scalar?

Reply:
Not at this moment

Reply:
For the headway inside the Transmodel's LegType, should I keep the headway in Int or any other suggestion?

Reply:
You can keep the Int.

Reply:
Noted